src/paho/mqtt/client.py

```diff
@@ -30,6 +31,7 @@
 import struct
 import threading
 import time
+import typing
```

Comment:
I would probably `from typing import ...` all of these to make things easier to read?

Reply:
Probably shouldn't import `typing` in two ways here either.